core/sqf/sqenvcom.sh

```diff
@@ -1,21 +1,18 @@
 # @@@ START COPYRIGHT @@@
 #
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
+# (C) Copyright 2007-2015 Hewlett-Packard Development Company, L.P.
```

Comment:
Looks like the new Apache copyright has been overlain with the old HP copyright. Please preserve the new Apache copyright.